app/controllers/staff/ratings_controller.rb

```diff
@@ -42,4 +43,10 @@ def update
   def rating_params
     params.require(:rating).permit(:score).merge(proposal: @proposal, user: current_user)
   end
+
+  def track_program_use
+    if params[:program]
+      enable_staff_program_subnav
```

Comment:
It's something to ponder in the future as maybe we don't allow rating in the program area and it's purely a review only function.  

In this case reviewers cannot submit ratings on their own talk as there's a before_action that stops it.  Program committee and organizers could hack the param to allow ratings but they can also just hop over to the program area to do the same thing so it's not an issue.